config/ftbquests/normal/chapters/17ca11c5/cfb77052.snbt

```diff
@@ -0,0 +1,16 @@
+{
+	title: "{q.bragging_rights.engineering_recipes.name}",
+	icon: "packagedexcrafting:basic_crafter",
+	x: 3.8d,
+	y: -3.2d,
+	text: [
+		"{q.bragging_rights.engineering_recipes.desc}"
+	],
+	dependencies: [
+		"b4b83088"
+	],
+	tasks: [{
+		uid: "2630fadd",
+		type: "checkmark"
+	}]
+}
```

Comment:
Making additional quest is bad idea. Especially without task and reward.

Please make "packaged Crafter" quest with ANY packaged crafting thing or something and suggest rewards that would looks good.
Follow this guide:
- Task should be item or one of many items (you can change "item list" option of the task)
- Reward should be fixed or with type "select reward". Selection is better. Try to find 3 items that would be awesome for progression. Remember, that it should not be items that skip other progression. Good examples is: tools, resources, crafting components, ingredients of more tools.

Reply:
The choice for extra quest is that I didn't want to rewrite "Gate: Extended Crafting" quest (while having the quests for 5x5, 7x7, 9x9 be basically nondescriptive); the icon is package crafting is because I thought 2 quests with same icon back to back looked silly.

Reply:
I can remove the new quest and just add the info from it into "Gate: Extended Crafting", if you want.

Reply:
Yes, this would be acceptable

Reply:
Done.